ui/app/helpers/utils/util.js

```diff
@@ -384,7 +384,7 @@ export function addHexPrefixToObjectValues (obj) {
  * Given the standard set of information about a transaction, returns a transaction properly formatted for
  * publishing via JSON RPC and web3
  *
- * @param {string|object|boolean} sendToken - Indicates whether or not the transaciton is a token transaction
+ * @param {boolean} [sendToken=undefined] - Indicates whether or not the transaciton is a token transaction
```

Comment:
Is this equivalent? Elsewhere when a parameter is sometimes omitted, we don't specify `undefined` as the default. I think `undefined` is implied to be the default if one isn't mentioned.
```suggestion
 * @param {boolean} [sendToken] - Indicates whether or not the transaciton is a token transaction
```